docs/core/compatibility/serialization/6.0/iasyncenumerable-serialization.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Breaking change: System.Text.Json IAsyncEnumerable serialization"
+description: Learn about the .NET 6.0 breaking change where the System.Text.Json supports IAsyncEnumerable serialization.
+ms.date: 10/01/2021
+---
+# System.Text.Json IAsyncEnumerable serialization
+
+<xref:System.Text.Json?displayProperty=fullName> now supports serializing and deserializing of
+<xref:System.Collections.Generic.IAsyncEnumerable%601> instances.
+
+## Previous behavior
+
+In previous versions, <xref:System.Text.Json?displayProperty=fullName> serialized <xref:System.Collections.Generic.IAsyncEnumerable%601> instances as empty JSON objects (`{}`). Deserialization failed with a <xref:System.Text.Json.JsonException>.
+
+## New behavior
+
+Asynchronous serialization methods now enumerate any <xref:System.Collections.Generic.IAsyncEnumerable%601> instances in an object graph and then serialize them as JSON arrays. Synchronous serializations methods do not support <xref:System.Collections.Generic.IAsyncEnumerable%601> serialization and throw a <xref:System.NotSupportedException>.
+
+## Version introduced
+
+6.0 Preview 4
+
+## Type of breaking change
+
+This change can affect [binary compatibility](../../categories.md#binary-compatibility).
```

Comment:
The change doesn't impact binary compatibility, it is a behavior change affecting the IAE serialization format.

Reply:
@eiriktsarpalis Does it affect source compatibility then?

"Source code may encounter a breaking change in behavior when targeting the new runtime/component/SDK, such as compile errors or different run-time behavior."

Reply:
The change impacts runtime serialization behavior, but there should be no compile-time errors or binary compatibility issues.